gas/config/tc-aarch64.c

```diff
@@ -10137,7 +10142,7 @@ aarch64elf_frob_symbol (symbolS * symp, int *puntp)
 const char *
 coff_aarch64_target_format (void)
 {
-  return "pe-aarch64-little";
+  return use_big_obj ? "pe-aarch64-bigobj" : "pe-aarch64-little";
```

Comment:
This usage of `use_big_obj` is not guarded by `#if defined (TE_PE) || defined (TE_PEP)`. Wouln't it lead to compilation error?

Reply:
It is guarded by OBJ_COFF and likely it works. Perhaps it makes sense to use OBJ_COFF for use_big_obj declaration.

Reply:
Changed all guarding to `OBJ_COFF` which seems to be used more in the `tc-aarch64.c`

Implementation tested here: https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/actions/runs/12292436394